docs/troubleshooting/common-issues.md

```diff
@@ -101,6 +101,24 @@ $ pex streamlit foo bar -o mypexbinary.pex
   $ ./mypexbinary.pex -m streamlit hello
   ```
 
+## Streamlit hangs when deployed as a web app
+
+If you're Streamlit app hangs when it's deployed as a web app (EC2, Heroku, AppEngine, CloudRun, etc.), you can try disabling CORS.
```

Comment:
We shouldn't propose that people disable CORS outright. Instead, the should tell people that the cause could be CORS and then tell them to try disabling CORS in order to diagnose the issue. If that fixes it, the best thing to do is to turn CORS back on and set `browser.serverPort`/`browser.serverAddress` to the URL and port of the app.

Reply:
Fixed.